src/snowflake/connector/file_transfer_agent.py

```diff
@@ -525,6 +525,14 @@ def _upload_files_in_sequential(self, file_metas: List["SnowflakeFileMeta"]):
         """
         idx = 0
         len_file_metas = len(file_metas)
+
+        cln_meta = file_metas[0].client_meta
+        client = cln_meta.storage_util.create_client(
+            cln_meta.stage_info,
+            use_accelerate_endpoint=cln_meta.use_accelerate_endpoint,
+        )
+        for meta in file_metas:
+            meta.client_meta.cloud_client = client
```

Comment:
Is this a regression? What caused this? can you please elaborate what the fix is?

Reply:
This is a regression. It was triggered by my last checkin I think which lost initializing client object code for this code path during a rebase.

Reply:
Thanks. Please update the JIRA.